src/melee/it/item.c

```diff
@@ -132,6 +132,17 @@ void HSD_JObjSetScaleItem(Item* it, HSD_JObj* jobj, Vec3* scl)
     HSD_JObjSetScale(jobj, scl);
 }
 
+/// @private
+#ifdef MUST_MATCH
+inline
+#endif
+    void
+    HSD_JObjSetFacingDirItem(HSD_JObj* jobj, Item* it)
+{
+    if (it->xDC8_word.flags.x19 == true)
+        HSD_JObjSetRotationY(jobj, M_PI / 2 * it->facing_dir);
+}
+
```

Comment:
I think this might be in `lb` or something because `Fighter` and `Item` both have their `facing_dir` at 0x2C.